src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -4436,13 +4438,32 @@ private static FamiliarData usableFamiliar(final Predicate<FamiliarData> familia
         .orElse(null);
   }
 
+  private static Set<Integer> graftedFamiliars() {
+    return Stream.of(
+            "zootGraftedButtCheekLeftFamiliar",
+            "zootGraftedButtCheekRightFamiliar",
+            "zootGraftedFootLeftFamiliar",
+            "zootGraftedFootRightFamiliar",
+            "zootGraftedHandLeftFamiliar",
+            "zootGraftedHandRightFamiliar",
+            "zootGraftedHeadFamiliar",
+            "zootGraftedNippleLeftFamiliar",
+            "zootGraftedNippleRightFamiliar",
+            "zootGraftedShoulderLeftFamiliar",
+            "zootGraftedShoulderRightFamiliar")
+        .map(pref -> Preferences.getInteger(pref))
+        .filter(id -> id > 0)
+        .collect(Collectors.toUnmodifiableSet());
```

Comment:
optional comment: returning the stream directly and doing `anyMatch`  feels better to me in some sense (probably perf or simplicity)